netty/src/main/java/io/grpc/netty/NettyServerHandler.java

```diff
@@ -213,6 +214,35 @@ static NettyServerHandler newHandler(
       boolean permitKeepAliveWithoutCalls,
       long permitKeepAliveTimeInNanos,
       Attributes eagAttributes) {
+    return newHandler(channelUnused, frameReader,frameWriter, transportListener,
+        streamTracerFactories, transportTracer, maxStreams, autoFlowControl, flowControlWindow,
+        maxHeaderListSize, maxMessageSize, keepAliveTimeInNanos, keepAliveTimeoutInNanos,
+        maxConnectionIdleInNanos, maxConnectionAgeInNanos, maxConnectionAgeGraceInNanos,
+        permitKeepAliveWithoutCalls, permitKeepAliveTimeInNanos, eagAttributes,
+        Ticker.systemTicker());
+  }
+
+  static NettyServerHandler newHandler(
```

Comment:
Remove this extra factory method and modify the existing one?

Reply:
done